code-challenges.test.js

```diff
@@ -14,19 +14,68 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("fibSeq", () => {
+  it("Returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    expect(fibSeq(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibSeq(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+  })
+  // FAIL  ./code-challenges.test.js
+  // fibSeq
+  //   ✕ eturns an array that length containing the numbers of the Fibonacci sequence
+
+  // ● fibSeq › eturns an array that length containing the numbers of the Fibonacci sequence
+
+  //   ReferenceError: fibSeq is not defined
+// })
+
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+// PseudoCode:
+// Input: A number
+// Output: returns an array that length containing the numbers of the Fibonacci sequence.
+// Create function called fibSeq
+// Using Array.from will allow us to create a new array the size of our number input
+// the .reduce method is going to return the accumulated sum while iterating through the array. 
+// since we are creating an array from scratch in the reduce field we name the array accum, leave the value absent and account for the index.
+// we then us the ternary to produce the outputs for each index using the formula for the fib sequence, Fn = Fn-1 + Fn-2.
+// The first if statement allows us to set the first two values in the array as [1,1] while the else statement executes the formula. 
+// allow empty brackets at the end as we created an empty array from the start.
+// no curly brackets or a return as the body of the function is a single expression. (implicit return)
```

Comment:
Interesting approach! Great logic and solid pseudo code! The explanation for the empty brackets at the end is a little off: that spot is used to specify what the accumulator is starting out as, and if left out, the accumulator will start out as whatever the first value in the array that's being reduced is.